app/reducers/gameState.js

```diff
@@ -58,15 +186,26 @@ const moveCardToPlayer = (state, cardId, playerId, target) => {
   return {
     ...sanitized,
     partyPool: sanitized.party.reduce((a, b) => a.concat(b), []),
-    [target]: sanitized[target].concat(cardId),
+    [target]: sanitized[target].concat(card),
   };
 };
 
 /**
  * move card from anywhere on the board to a shared target pile
  */
-const moveCard = (state, id, target) => {
-  const newState = { players: state.players };
+const moveCard = (
+  state,
+  id,
+  target?: 'foodDeck' | 'guestDeck' | 'guestDiscard' | 'aura' | 'trash',
+) => {
+  const newState: State = {
+    ...state,
+    players: state.players,
+    foodDeck: [],
+    guestDeck: [],
+    guestDiscard: [],
+    aura: [],
```

Comment:
Might this be where the shared states are disappearing?

Reply:
😡  Perhaps; the subsequent code is supposed to repopulate them with the contents of `state`, filtered for the card that we're moving. Following the mutation is a little difficult, but I'll add a test to be sure; hopefully that'll make it obvious.

Reply:
Fortunately unit test says that's not it, so at least I'm not crazy about this particular function. Going hunting.

Reply:
Oh Jesus Christ; how did that ever work? Fixing. I need to either be less stupid or more relaxed about being stupid for the sake of my blood pressure.